client/package.json

```diff
@@ -0,0 +1,17 @@
+{
+    "name": "@linera/client",
+    "version": "1.0.0",
+    "description": "",
+    "main": "pkg/linera_web.js",
+    "types": "pkg/linera_web.d.ts",
+    "scripts": {
+        "install": "true",
+        "build": "bash build.bash",
+        "prepare": "pnpm build",
+        "lint": "cargo fmt --check && cargo clippy -- -W clippy::pedantic",
+        "ci": "pnpm lint"
+    },
+    "keywords": [],
+    "author": "",
+    "license": "ISC"
```

Comment:
Should we populate this, or are we planning on doing that later?

Reply:
No time like the present!